# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmTlsData.h
# Commit: 40f861f62a0f
# Full Hash: 40f861f62a0f58040cb6cadf073bc5e97a44231d
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2021-12-23 09:34:36
# Regressor Bug: 1642412
# File Overlap Count: 7
# Description:
#   Bug 1642412 - wasm: Optimize baseline table.size/get/set. r=lth
#   
#   This commit adds a fast path for table.get/set when the table
#   representation is anyref, and an unconditional fast path for
#   table.length.
# ==============================================================================

diff -r 58a45475b524 -r 40f861f62a0f js/src/wasm/WasmTlsData.h
--- a/js/src/wasm/WasmTlsData.h	Wed Dec 22 19:09:54 2021 +0200
+++ b/js/src/wasm/WasmTlsData.h	Wed Dec 22 17:36:29 2021 +0000
@@ -166,7 +166,7 @@
 
   // Pointer to the array of elements (which can have various representations).
   // For tables of anyref this is null.
-  void* functionBase;
+  void* elements;
 };
 
 // Table element for TableRepr::Func which carries both the code pointer and
